R/dt_heading.R

```diff
@@ -49,5 +49,9 @@ dt_heading_has_subtitle <- function(data) {
 
   heading <- dt_heading_get(data = data)
 
-  length(heading) > 0 && length(heading$subtitle) > 0 && !is.null(heading$subtitle)
+  length(heading) > 0 &&
+    length(heading$subtitle) > 0 &&
+    !is.null(heading$subtitle) &&
+    heading$subtitle != "" &&
+    !grepl("^\\s+$", heading$subtitle)
```

Comment:
```suggestion
    length(heading$subtitle) > 0 &&
    any(grepl("\\S", heading$subtitle))
```

Reply:
I added the function `is_nonempty_string()` with `length(x) > 0 && any(grepl("\\S", x))` in the body. Now a majority of the `&&` chain is simplified here.